Main/tests/GoodWalkerTest.kt

```diff
@@ -0,0 +1,12 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class GoodWalkerTest {
+    @Test
+    fun `should return string with GoodWalker`(){
+        val controlSpeed = 5
+        val actual = GoodWalker()
```

Comment:
Actual must be final. But u'r use in the assert actual.walk 
It's bad practice 
Move calling method to actual initialisation